joomla/joomla-cms
# [fix] rgt root tag value not updated on sample_testing

#### Steps to reproduce the issue
1. Install a joomla with Test English sample data.
2. Now go to backend and create a new tag (for example `test`) with no parent tag. You will see that the tag path assigned to the tag is wrong: `red/test`

I detected the issue trying to assign my new tag to any article. When writing `test` in the tags field the wrong path will be suggested.

This is caused because the ROOT tag already exists and then the insert on sample data is ignored. So the `rgt` value is never updated. We need to update the ROOT `rgt` value after inserting the other tags.
#### Expected result

A tag with no parent shouldn't have any parent in the path. In our example a `test` tag with no parent should have a `test` path.
#### System information (as much as possible)

Latest Joomla (v3.4) from staging branch.

```
$ php --version
PHP 5.5.9-1ubuntu4.6 (cli) (built: Feb 13 2015 19:17:11) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

```
$ mysql --version
mysql  Ver 14.14 Distrib 5.5.41, for debian-linux-gnu (x86_64) using readline 6.3
```
#### Backward compatibility

This patch only affects to the sample data so it cannot break any existing site.
#### How to test patch

Apply the patch and then clean `configuration.php` contents to create a new installation with the Test English sample data. Then create again the `test` tag and see if the `path` value is correct.
#### Additional comments

My patches for postgreSQL and SQL server are blindly applied so ideally we should have someone testing them.
